pkgs/by-name/cu/cutentr/package.nix

```
@@ -0,0 +1,66 @@
+{
+  stdenv,
+  libsForQt5,
+  fetchFromGitLab,
+  makeDesktopItem,
+  lib,
+  copyDesktopItems,
+}:
+
+let
+  version = "0.3.3";
+in
+
+stdenv.mkDerivation {
+  pname = "cutentr";
+  inherit version;
+
+  src = fetchFromGitLab {
+    owner = "BoltsJ";
+    repo = "cuteNTR";
+    tag = version;
```

using finalAttrs here would be clear and make the pkg better respond to being overridden 
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "cutentr";
  version = "0.3.3";

  src = fetchFromGitLab {
    owner = "BoltsJ";
    repo = "cuteNTR";
    tag = finalAttrs.version;
```